.eslintrc.js

```diff
@@ -170,32 +170,31 @@ module.exports = {
         message: '__defineSetter__ is deprecated.',
       },
     ],
-    // If this list is modified, please copy the change to lib/.eslintrc.yaml
-    // and test/.eslintrc.yaml.
+    // If this list is modified, please copy changes that should apply to ./lib as well to lib/.eslintrc.yaml.
```

Comment:
Even though the `max-len` rule is disabled earlier in the file, I'd prefer we wrap this comment at 80 characters.